crates/core/src/loader.rs

```diff
@@ -58,6 +61,18 @@ pub fn locate_tool(id: &Id, proto: &ProtoEnvironment) -> Result<PluginLocator, P
         }
     }
 
+    // Search in registries
+    if locator.is_none() {
```

Comment:
Let's remove this, since nothing by default is in a registry.

This functionality should also be reserved for the official proto registry.

Reply:
Can we agree to change the condition to `if locator.is_none() && config.settings.registries.is_some() {`

Because this would give us the possibility to define our company registry and take all plugins from there.

```toml
java = "8"
gradle = "latest"
conan = "latest"

[settings]
registries = [
   { "registry" = "REGISTRY", "organization" = "ORG" }
]

```



Otherwise, we would still have to define it in this way

```toml
java = "8"
gradle = "latest"
conan = "latest"

[plugins]
java = "registry://REGISTRY/ORG/java:latest"
gradle = "registry://REGISTRY/ORG/java:latest"
conan = "registry://REGISTRY/ORG/java:latest"

```